partner_event/views/event_registration_view.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+     Copyright 2016 Vicent Cubells <vicent.cubells@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+<record id="view_registration_search" model="ir.ui.view">
+    <field name="name">Search attended registrations</field>
+    <field name="model">event.registration</field>
+    <field name="inherit_id" ref="event.view_registration_search"/>
+    <field name="arch" type="xml">
+        <filter name="open" position="after">
```

Comment:
Seems like [they changed it to `expected`](https://github.com/odoo/odoo/blob/10.0/addons/event/views/event_views.xml#L391).